src/components/style/index.css

```diff
@@ -18,8 +18,7 @@
 @import './cornerPosition.css';
 @import './countdownButton.css';
 @import './dateInput.css';
-@import './radio.css';
```

Comment:
is there a reason to remove these imports? I checked and we have some components using these styles

Reply:
The imports I removed were duplicates.